core/storage/email/gae_models.py

```diff
@@ -97,13 +106,15 @@ def get_deletion_policy():
 
     @staticmethod
     def get_model_association_to_user():
+        # type: () -> base_models.MODEL_ASSOCIATION_TO_USER
         """Users already have access to this data since emails were sent
         to them.
         """
         return base_models.MODEL_ASSOCIATION_TO_USER.NOT_CORRESPONDING_TO_USER
 
     @classmethod
     def get_export_policy(cls):
+        # type: () -> Dict[Text, base_models.EXPORT_POLICY]
```

Comment:
We're now in Python 3. Could you use str throughout instead? Would be better not to promulgate Text all over the place if that's not needed.

Reply:
Yes, we should do this. Will also remove this point from the wiki. Thanks!